packages/web-components/src/components/card-in-card/__stories__/card-in-card.stories.ts

```diff
@@ -35,13 +36,32 @@ export const Default = ({ parameters }) => {
       </dds-card-in-card-image>
       <dds-card-eyebrow>${eyebrow}</dds-card-eyebrow>
       <dds-card-heading>${heading}</dds-card-heading>
-      <dds-card-in-card-footer>
+      <dds-card-cta-footer>
         ${ArrowRight20({ slot: 'icon' })}
-      </dds-card-in-card-footer>
+      </dds-card-cta-footer>
```

Comment:
If this is a minor breaking change we should note this in the release notes. Not sure if AEM in particular is already using this or not.

Reply:
oh good call! agreed!